downstream/modules/platform/proc-controller-github-app-token.adoc

```diff
@@ -12,24 +12,32 @@ For more information, see link:https://docs.github.com/en/apps/creating-github-a
 . Create a lookup credential that stores your secrets. 
 For more information, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/using_automation_execution/controller-credentials#controller-create-credential[Creating new credentials].
 
-. Select *GitHub App Installation Access Token lookup* for *Credential type*, and enter the following attributes to properly configure your lookup:
+. Select *GitHub App Installation Access Token Lookup* for *Credential type*, and enter the following attributes to properly configure your lookup:
 
-** *GitHub App ID*: Enter the app ID used for communicating with your GitHub app.
-** *GitHub App Installation ID*: Enter the ID of the installation that you want to authenticate as.
-** *RSA Private Key*: Enter the generated private key from the GitHub organization in your repository. 
+** *GitHub App ID*: Enter the App ID provided by your instance of GitHub, this is what is used to authenticate.
+** *GitHub App Installation ID*: Enter the ID of the application into your target organization that the access token is scoped to. 
+You must set it up to have access to your target repository.
```

Comment:
what needs to be set up here? the application, the organization of the token?

Reply:
The context isn't in this diff, but this step sets up the lookup credential itself. The dependency is setting up the application but that's documented in a github doc, which is also already referenced in a prior PR.